thunder/torch/__init__.py

```diff
@@ -2842,6 +2842,19 @@ def _normalize(a: TensorProxy, /, norm_dims, eps: Number) -> tuple[TensorLike, T
     return out, mean, rstd
 
 
+@torchsymbol(torch.nn.functional.normalize, is_method=True)
+def normalize(
+    a: TensorProxy, /, p: float = 2, dim: int | Sequence[int] = 1, eps: float = 1e-12, out: None | TensorProxy = None
+) -> TensorProxy:
+    utils.check(out is None, lambda: "normalize: out is not None which is currently unsupported", NotImplementedError)
+    denom = pow(a, p)
```

Comment:
This only works for finite p, PyTorch also takes inf. Could you add `p == math.inf` + tests for maybe p=1, 2, 4, inf , please?

Reply:
nit: `a**p` instead of `pow` is probably more pythonic

Reply:
Oh, I wasn't aware of `inf`. Thanks for catching! Will test these!
+ Thanks for suggesting a pythonic approach Mark!